src/block/block_open.c

```diff
@@ -194,41 +196,23 @@ __wt_block_open(WT_SESSION_IMPL *session,
 	/* Configuration: optional OS buffer cache maximum size. */
 	WT_ERR(__wt_config_gets(session, cfg, "os_cache_max", &cval));
 	block->os_cache_max = (size_t)cval.val;
-#ifdef HAVE_POSIX_FADVISE
```

Comment:
Why did you decide to drop this error checking? If we are going to start silently ignoring this, we should probably update our documentation to reflect that this is an advisory setting.